test/block_test.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'test_helper'
+
+describe "Block class" do 
+  before do 
+    @reservation_maker = Hotel::Reservation_Maker.new 
+    @check_in = Date.new(2020, 01, 01)
+    @check_out = @check_in + 4
+    @date_range = Hotel::Date_Range.new(@check_in, @check_out)
+    @block_rooms = @reservation_maker.rooms[0..3]
+    @discount = 150
+
+    @block = Hotel::Block.new(@date_range, @block_rooms, @discount)
+    @room = @block_rooms[0]
+  end 
+
+  describe "initialize" do
+    it "is an instance of Block" do 
+      expect(@block).must_be_kind_of Hotel::Block
+    end 
+  end 
+  
+  describe "add block to list of blocks" do
+    it "returns a list of all blocks" do
+      block_array = @reservation_maker.add_block(@block)
```

Comment:
Because this Act step is about a method that exists in the `ReservationMaker` class (aka `@reservation_maker.add_block`), I would probably move this test to the `ReservationMaker` test!